appeals/web/src/server/appeals/appeal-documents/appeal-documents.controller.js

```diff
@@ -14,8 +14,10 @@ import {
 	manageFolderPage,
 	manageDocumentPage,
 	changeDocumentDetailsPage,
+	ChangeDocumentFilenamePage,
```

Comment:
Minor one, could this function name be changed to camelCase?

Reply:
Done as requested

Reply:
Sorry, I meant it shouldn't start with a capital - could it be changed to `changeDocumentFilenamePage` or `changeDocumentFileNamePage`?

Reply:
Ahh, got it.  Fixed